app/bundles/EmailBundle/Swiftmailer/Momentum/DTO/TransmissionDTO/RecipientDTO/AddressDTO.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Mautic\EmailBundle\Swiftmailer\Momentum\DTO\TransmissionDTO\RecipientDTO;
+
+/**
+ * Class AddressDTO.
+ */
+final class AddressDTO implements \JsonSerializable
+{
+    /**
+     * @var string
+     */
+    private $email;
+
+    /**
+     * @var string
+     */
+    private $name;
+
+    /**
+     * @var string|null
+     */
+    private $headerTo = null;
+
+    /**
+     * AddressDTO constructor.
+     *
+     * @param string $email
+     * @param string $name
+     * @param bool   $bcc
+     */
+    public function __construct($email, $name, $bcc = false)
+    {
+        $this->email = $email;
+        $this->name  = $name;
+        if ($bcc === false) {
+            $this->headerTo = $email;
```

Comment:
I want only to confirm this - Do you add a separate `AddressDTO` for each BCC? This means that you provide an email address in second argument only for a BCC if third argument is set, correct?

Reply:
Each BCC is separate instance of AddressDTO created as new AddressDTO('bcc@email.com', 'Bcc Name', true); 

I do not understand providing email in second argument. Second argument is name. What do you mean?

Reply:
Yes, I meant first argument. Sorry.